FIX: arch-mage grants magic+2 (instead of +1)

Murphy812 opened the pull request:
Fixed: passive ability 'arch-mage' should grants +2 to Magic attribute instead of +1.

https://trello.com/c/2Ircc8vA/861-%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D1%82%D1%8C-%D0%B1%D0%BE%D0%BD%D1%83%D1%81-%D0%BE%D1%82-%D0%B0%D1%80%D1%85%D0%B5%D1%82%D0%B8%D0%BF%D0%B0-%D0%BC%D0%B0%D0%B3-2-%D0%BA-%D0%BC%D0%B0%D0%B3%D0%B8%D0%B8

Murphy812 committed:
FIX: arch-mage grants magic+2 (instead of +1)

Fixed: passive ability 'arch-mage' should grants +2 to Magic attribute instead of +1.

aeremin commented:
Мне кажется, твой тест для Aurma какой-то кривой/нестабильный. Он падает тут и падал в каком-то из пулл-реквестов Кривды. Можешь глянуть?

Murphy812 committed:
FIX: aura chance change

Fixed: new random character of aura will never be the same as old one. So test for Aurma will get precise number of changed characters

Murphy812 commented:
> Мне кажется, твой тест для Aurma какой-то кривой/нестабильный. Он падает тут и падал в каком-то из пулл-реквестов Кривды. Можешь глянуть?

Там смешнее ;) символ a-z менялся на случайный символ из a-z. А в тесте проверялось точное число изменённых символов.
